src/main/java/org/jboss/pnc/api/deliverablesanalyzer/dto/WindowsArtifact.java

```diff
@@ -0,0 +1,70 @@
+/**
+ * JBoss, Home of Professional Open Source.
+ * Copyright 2014-2020 Red Hat, Inc., and individual contributors
+ * as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jboss.pnc.api.deliverablesanalyzer.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotEmpty;
+import java.util.List;
+
+/**
+ *
+ * @author dwalluck
+ */
+@Data
+@SuperBuilder
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+@Jacksonized
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public class WindowsArtifact extends Artifact {
```

Comment:
Prefer `Windows` or `Win`? Type constant from Koji is `win`.

Reply:
Windows is fine

Reply:
Yes. I looked at the code again, and I see that for Maven artifcats it spells out "Maven" and is not using an abbreviation like "Mvn", so I think the full name better matches the existing code.

Reply:
@janinko Wait, should `WIN` in `ArtifactType` also be changed to `WINDOWS`?

Reply:
I made the change for consistency. It will be called `WINDOWS` in pnc-api and `WIN` in kojiji.